### PR TITLE
LRQA- 81128 - Automation SaveImportCustomObjectAsTemplate

### PR DESCRIPTION
**Pull request by pedroamarall**
cc: @manoelcyreno

### CODE REVIEW

**Comment by pedroamarall** [comment]
ci:test:sf

File: modules/apps/batch-planner/batch-planner-test/src/testFunctional/tests/ImportAndExportObjects.testcase [hunk 1]
```diff
@@ -245,6 +245,39 @@ definition {
...
+				en_US_plural_label = "students",
+				name = "Student",
+				requiredStringFieldName = "name");
+		}
+
+		task ("And Given I create an import task with custom objects") {
+			ImportExport.openImportExportAdmin();
+
+			ImportExport.importFile(
+				entityType = "Account (v1_0 - Liferay Headless Admin User)",
```
Comment (magjed4289) [inline modules/apps/batch-planner/batch-planner-test/src/testFunctional/tests/ImportAndExportObjects.testcase:262]: The scenario is saying to save the template with a custom object, it this case Student. If it is not possible to do it for now with Student, we can leave the test prepared and @ignore it until the issue is fixed

**Review by magjed4289 — CHANGES_REQUESTED** [review]

File: modules/apps/batch-planner/batch-planner-test/src/testFunctional/macros/ImportExport.macro [hunk 2]
```diff
@@ -367,9 +367,13 @@ definition {
 
 		ImportExport.mapImport(fieldMappings = ${fieldMappings});
 
-		ImportExport.startImportAndWaitForSuccess();
+		if (!(isSet(noexport))) {
```
[the remark this answers is not in the document]
Comment (pedroamarall) [reply modules/apps/batch-planner/batch-planner-test/src/testFunctional/macros/ImportExport.macro]: Done changes, thanks!

File: modules/apps/batch-planner/batch-planner-test/src/testFunctional/tests/ImportAndExportObjects.testcase (hunk 1 above)
Comment (pedroamarall) [reply modules/apps/batch-planner/batch-planner-test/src/testFunctional/tests/ImportAndExportObjects.testcase]: Done changes, thanks!

**Comment by pedroamarall** [comment]
ci:test:sf

**Review by magjed4289 — CHANGES_REQUESTED** [review]

File: modules/apps/batch-planner/batch-planner-test/src/testFunctional/tests/ImportAndExportObjects.testcase [hunk 3]
```diff
@@ -273,6 +273,37 @@ definition {
...
+				en_US_plural_label = "students",
+				name = "Student",
+				requiredStringFieldName = "name");
+
+			ImportExport.openImportExportAdmin();
+
+			ImportExport.importFile(
+				entityType = "C_Student (v1_0 - Liferay Object REST)",
+				fieldMappings = "name:Account Name,externalReferenceCode:External Reference Code",
+				fileName = "csv_account_import.csv",
```
[the remark this answers is not in the document]
Comment (pedroamarall) [reply modules/apps/batch-planner/batch-planner-test/src/testFunctional/tests/ImportAndExportObjects.testcase]: Added an ignore for finding a bug, reported it in ticket [LPS-18910](https://liferay.atlassian.net/browse/LPS-189103)

**Comment by pedroamarall** [comment]
ci:test:sf

**Review by magjed4289 — APPROVED** [review]

**Comment by magjed4289** [comment]
ci:forward

**Pull request closed without merging**